app-misc/pet/pet-0.2.4.ebuild

```diff
@@ -20,13 +20,11 @@ EGO_VENDOR=( "github.com/BurntSushi/toml b26d9c308763d68093482582cea63d69be07a0f
 
```

Comment:
Please improve the summary of commit message.

1. Bug numbers require referencing external sources, please instead focus on what you're doing.
2. I don't think 650096 is somewhat 'most important' part of this commit. It looks like a major cleanup instead.
3. I think it would be better to do this in a series of split commits for every logical change.